core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -122,7 +122,7 @@ public static synchronized CoreSocketFactory getInstance() {
   static ListeningScheduledExecutorService getDefaultExecutor() {
     // TODO(kvg): Figure out correct way to determine number of threads
     ScheduledThreadPoolExecutor executor =
-        (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(2);
+        (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(8);
```

Comment:
Why 8? Do you think there's a better default value?

Reply:
I added a comment with the rationale.

Reply:
LGTM. Thanks.